plugins/database/src/db_plugin.cpp

```diff
@@ -232,6 +241,85 @@ namespace
 
         return SUCCESS();
     } // get_auth_config
+
+    auto get_token_lifetime_configuration(std::int32_t& _out) -> irods::error
+    {
+        constexpr auto default_token_lifetime_in_seconds = 1209600; // 2 weeks
+        constexpr const char* config_namespace = "authentication";
+        constexpr const char* config_option_name = "token_lifetime_in_seconds";
+
+        // 12 is one more than number of digits for max int32 which will prevent erroneous truncations and detect
+        // overflows and underflows, plus null terminator.
```

Comment:
```
... for max int32 ...
```

Why is this tripping me up? Can you explain this statement?

I take it it's talking about epoch time and null bytes?